arm-network/2016-09-01/swagger/routeFilter.json

```diff
@@ -0,0 +1,849 @@
+            "type": "string",
+            "description": "Expands referenced resources."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting Route Filter resource.",
+            "schema": {
+              "$ref": "#/definitions/RouteFilter"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "RouteFilters"
+        ],
+        "operationId": "RouteFilters_CreateOrUpdate",
+        "description": "Create or updates a route filter in a specified resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "routeFilterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the route filter."
+          },
+          {
+            "name": "routeFilterParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RouteFilter"
+            },
+            "description": "Parameters supplied to the create or update route filter operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting Route Filter resource.",
+            "schema": {
+              "$ref": "#/definitions/RouteFilter"
+            }
+          },
+          "201": {
+            "description": "Create successful. The operation returns the resulting Route Filter resource.",
+            "schema": {
+              "$ref": "#/definitions/RouteFilter"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+          "RouteFilters"
+        ],
+        "operationId": "RouteFilters_Update",
+        "description": "Updates a route filter in a specified resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "routeFilterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the route filter."
+          },
+          {
+            "name": "routeFilterParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RouteFilter"
```

Comment:
Request body for a PATCH operation cannot have required properties at the top level and any property (nested too) cannot have default values. Please create a new model type as an input for PATCH.

Btw, does the PATCH only supports patching tags or is a full blown patch?